### ISSUE TITLE
autoreview with possibility to change default answer:again, and hand-free autoreview when screen off

### ISSUE THREAD

**Issue by hyper-hub** [ankidroid/Anki-Android#5112]
hello.   question or feature request.  

in ankidroid version, 
when tts enable in settings.,and when set automatic display answer, and set time to show next question. , then review started in automatic mode,. with speech text in card, 
In this mode it use "again" button for all auto-reviewed card, but this more better if it will be a possibility to change - again answer,  to one of the other button  (i.e hard, good, easy) or set custom interval answer for all auto-reviewed card (for example 50 days or any other). And all auto-reviewed card, answered according to chosen button. 
and all this need to work when display turned off., and anki still continue auto-review like audioplayer app,  
but for now, when screen turned-off, then auto-review is stopped.  
i think this feature will be useful -  for review in this matter when walking, running or traveling, ie  free-hand reviewing with screen-off.,  listening tts speech via headphone, Bluetooth etc.  

thanks.

**Comment by mikehardy** [comment]
The request for an option to change what response is selected on auto-review is a duplicate of #2609 - it has been around a while but is fairly popular. You should subscribe to that issue to be notified of progress

I will alter the title to be for the part about continuing to operate in the background

You may find the ability to use headset volume control buttons to answer cards useful https://github.com/ankidroid/Anki-Android/issues/3021#issuecomment-160243369

**Comment by p4nix** [comment]
A copy from the thing I wrote in #6563 requesting a similar feature:

A lot of people have cards that do not actually require you to see the screen. This especially applies to basic q&a cards, cloze cards and audio language learning cards.
<hr>

It would be a nice option to be able to review those cards and grade them via:

- a bluetooth controller
- bluetooth audio volume controls
- volume controls of the device

while the screen is turned off and get the questions and answers read through the TTS engine (without the requirement of using tags).
<hr>

I'm not myself a developer, but here are some thoughts about the additional requirements this may have:
- ideally, be able to turn on and off this mode via the settings. Some people might not like this option as they don't want to accidentally review cards when they turned off their screen without exiting AnkiDroid.
 -> also, this could automatically turn off the auto-answer mode and vice versa, as they might lead to interesting and dangerous combinations ;)
- define an option to either white list or black list cards which can be reviewed in this mode, probably based on a tag. Those tags should ideally be defined by the user rather than hardcoded.
- add an option to ignore cards with MathJax/images/other content not suitable for turned off screens, which can be overwritten by the whitelist tag.
- for cards including sounds, add an option to not pass the question text through the TTS engine as only the audio might be needed.

**Comment by mikehardy** [comment]
No, AnkiDroid is not going to be adding screen off functionality, I've worked on other apps that do things while the screen is off and it's a whole 'nother thing.
